doc/classes/GraphNode.xml

```diff
@@ -25,86 +34,72 @@
 				Disables input and output slot whose index is [param slot_index].
 			</description>
 		</method>
-		<method name="get_connection_input_color">
+		<method name="get_port_input_color">
 			<return type="Color" />
-			<param index="0" name="port" type="int" />
+			<param index="0" name="port_idx" type="int" />
 			<description>
-				Returns the [Color] of the input connection [param port].
+				Returns the [Color] of the input connection [param port_idx].
```

Comment:
```suggestion
				Returns the [Color] of the input port for the given [param port_idx].
```

Reply:
I would use "with the given" instead of "for the given".

Reply:
Sure!